assets/js/base/components/label/index.js

```diff
@@ -48,13 +54,12 @@ Label.propTypes = {
 	screenReaderLabel: PropTypes.string,
 	wrapperElement: PropTypes.oneOfType( [
 		PropTypes.string,
-		elementType,
+		PropTypes.elementType,
```

Comment:
Looking into this, it looks like under the hood, `PropTypes.elementType` is using [react-is](https://www.npmjs.com/package/react-is).  So this type also validates string elements (like 'div', 'span' etc).  So it should be safe to just remove the `PropTypes.string` here and only validate `PropTypes.elementType`.

Reply:
Good catch! Changed in 1fcd28aa27f99e7cfbc6795cd8c065517aac2eb9.